test/extended/builds/run_policy.go

```diff
@@ -145,7 +145,7 @@ var _ = g.Describe("[builds][Slow] using build configuration runPolicy", func()
 				}
 				if build.Status.Phase == buildapi.BuildPhaseRunning {
 					latency := lastCompletion.Sub(time.Now())
-					o.Expect(latency).To(o.BeNumerically("<", 10*time.Second), "next build should have started less than 10s after last completed build")
+					o.Expect(latency).To(o.BeNumerically("<", 15*time.Second), "next build should have started less than 15s after last completed build")
```

Comment:
yeah, running in parallel seems to have thrown the timing off a bit by slowing things down.